types/nise/index.d.ts

```diff
@@ -0,0 +1,291 @@
+        /**
+         * Responds to all method requests to URLs matching the regular expression.
+         */
+        respondWith(method: string, url: RegExp, body: string): void;
+        /**
+         * Responds to all method requests to URLs matching the regular expression.
+         */
+        respondWith(method: string, url: RegExp, response: any[]): void;
+        /**
+         * Responds to all method requests to URLs matching the regular expression.
+         */
+        respondWith(method: string, url: RegExp, fn: (xhr: FakeXMLHttpRequest) => void): void;
+        respondWith(...args: any[]): void;
+
+        /**
+         * Causes all queued asynchronous requests to receive a response.
+         * If none of the responses added through respondWith match, the default response is [404, {}, ""].
+         * Synchronous requests are responded to immediately, so make sure to call respondWith upfront.
+         * If called with arguments, respondWith will be called with those arguments before responding to requests.
+         */
+        respond(): void;
+        restore(): void;
+
+        getRequest(): FakeXMLHttpRequest | undefined;
+        reset(): void;
+        resetBehavior(): void;
+        resetHistory(): void;
+    }
+
+    interface FakeXMLHttpRequestStatic {
+        new(): FakeXMLHttpRequest;
+        /**
+         * Default false.
+         * When set to true, Sinon will check added filters if certain requests should be “unfaked”
+         */
+        useFilters: boolean;
+        /**
+         * Add a filter that will decide whether or not to fake a request.
+         * The filter will be called when xhr.open is called, with the exact same arguments (method, url, async, username, password).
+         * If the filter returns true, the request will not be faked.
+         * @param filter
+         */
+        addFilter(filter: (method: string, url: string, async: boolean, username: string, password: string) => boolean): void;
+        /**
+         * By assigning a function to the onCreate property of the returned object from useFakeXMLHttpRequest()
+         * you can subscribe to newly created FakeXMLHttpRequest objects. See below for the fake xhr object API.
+         * Using this observer means you can still reach objects created by e.g. jQuery.ajax (or other abstractions/frameworks).
+         * @param xhr
+         */
+        onCreate(xhr: FakeXMLHttpRequest): void;
+        /**
+         * Restore original function(s).
+         */
+        restore(): void;
+    }
+
+    interface FakeServerStatic {
+        create(options?: Partial<FakeServerOptions>): FakeServer;
+    }
+
+    interface FakeXHR {
+        useFakeXMLHttpRequest(): FakeXMLHttpRequestStatic;
+        FakeXMLHttpRequest: FakeXMLHttpRequestStatic;
+    }
+
+    interface API {
+        fakeServerWithClock: FakeServerStatic;
+        fakeServer: FakeServerStatic;
+        fakeXhr: FakeXHR;
+    }
+}
+
+declare const Nise: Nise.API;
```

Comment:
Rather than declaring this const and merging it with the namespace `Nise`, it would be cleaner to just do the following:
```ts
export const fakeServerWithClock: FakeServerStatic;
export const fakeServer: FakeServerStatic;
export const fakeXhr: FakeXHR;
```
and above that, just define each of the interfaces you want to use and export those that you want consumers of this types definition package to be able to reference directly.

You can use [this template](http://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html) as a guideline.